crates/router/src/core/payments/helpers.rs

```diff
@@ -984,6 +990,7 @@ pub async fn create_temp_card(
         .insert_temp_card(temp_card)
         .await
         .change_context(errors::ApiErrorResponse::InternalServerError)
+        .attach_printable("Failed while inserting card to temp card") // Deprecated
```

Comment:
You can ignore this, I'm removing `temp_card` in #366.

Reply:
Sure!